### DISCUSSION TITLE
运行错误

### DISCUSSION THREAD

**Discussion by jinzezhi** [hktalent/scan4all#48] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/93123422/182020998-016dbf23-479f-4aba-92dc-0023d394261f.png)

**Comment by jinzezhi** [comment]
看上去像是调用nmap时出错

**Reply by hktalent** [reply]
自己外面运行nmap得到结果xxx.xml
./scan4all -l xxx.xml -v
即可

**Comment by jinzezhi** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/93123422/182022053-3e2c054f-545b-4515-94a1-1680032ce840.png)
我把呀运行的nmap命令单独拿出来，判断出了错误点

**Reply by hktalent** [reply]
你可以自己修改当前目录中config/config.json
--unique是去重用的，或者升级你的nmap版本

**Reply by jinzezhi** [reply]
oo 知道了 我发现核模板没有进行验证
![image](https://user-images.githubusercontent.com/93123422/182022286-4fc3cebc-b26a-4573-ab87-31fb1dda109b.png)

**Reply by hktalent** [reply]
config.json
中设置
"enablEmbedYaml": true,

或命令行前
enablEmbedYaml=true ./scan4all -host xxx.com

**Reply by hktalent** [reply]
你可以自己修改当前目录中config/config.json
--unique是去重用的，或者升级你的nmap版本
config.json
中设置
"enablEmbedYaml": true,

或命令行前
enablEmbedYaml=true ./scan4all -host xxx.com

**Reply by jinzezhi** [reply]
没错的，配置文件中是true

**Reply by jinzezhi** [reply]
![image](https://user-images.githubusercontent.com/93123422/182022720-b65bb931-7944-46c0-ade6-a036ff9c2603.png)
这是目前全部的，已经卡死在这里了

**Reply by jinzezhi** [reply]
可以加个微信吗 tna5757  这是我的微信

**Reply by hktalent** [reply]
没有卡死，在后台爆破密码中，下载最新版本
-v
就可以看到进度
"hydrathread": 64,
设置大一点，加快爆破速度

**Reply by jinzezhi** [reply]
下载hydra最新版？

**Reply by hktalent** [reply]
最新版本的scan4all
